tcms/telemetry/static/telemetry/js/testing/execution-trends.js

```diff
@@ -97,7 +99,13 @@ function drawChart() {
                 value: (value, _ratio, _id, _index) => value ? value : undefined
             }
         }
+        config.legend = {
+            hide: [totalKey],
+        }
         c3.generate(config);
+
+        // hide the total data point
+        $(`.c3-target-${totalKey}`).addClass('hidden')
```

Comment:
legend is still shown, TOTAL line chart is also shown.

Reply:
how about now?